app/javascript/react/components/ThresholdConfigurator/ThresholdConfigurator.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useEffect, useRef, useState } from "react";
+
+import * as S from "./ThresholdConfigurator.style";
+
+const calculateThumbPercentage = (
+  value: number,
+  min: number,
+  max: number
+): number => {
+  const percentage = (value - min) / (max - min);
```

Comment:
Did you consider the case where min is negative number? F.e. temperature can be negative and I believe we measure temp.

Reply:
Yes, it should work the same. :) I was adding test for it, but struggled with jest setup. I will add jest setup as a task on our board.